dali/dfu/dfuutil.cpp

```diff
@@ -858,9 +858,38 @@ class CDFUhelper: public CInterface, implements IDFUhelper
                 throwError1(DFUERR_DFileNotFound, lfn);
         }
         else {
-            Owned<IFileDescriptor> fdesc = file->getFileDescriptor();
-            Owned<IPropertyTree> t = fdesc->getFileTree();
-            toXML(t, out, true);
+            CDfsLogicalFileName tmplfn;
+            StringBuffer lfnpath;
+            bool safe = false;
+            if ((strstr(lfn, "::") != NULL) && !strchr(lfn, '/'))
```

Comment:
I may have been mistaken, but looking again at the server-side implementation of getFileTree() - it still looks like it will serialize the SuperFile meta only.

Can you check and debug what it's actually doing for a super file request?

If there is no way to do it, then a way via IDistributedFileDirectory should be added.